x/concentrated-liquidity/tick_test.go

```diff
@@ -0,0 +1,70 @@
+package concentrated_liquidity
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func Test(t *testing.T) {
```

Comment:
```suggestion
func TestNextInitializedTick(t *testing.T) {
```

We also have test suite set up on main, feel free to use that as well!